docs/pages/installation.mdx

```diff
@@ -86,11 +86,11 @@ docker pull quay.io/gravitational/teleport:(=teleport.version=)
 
 ## Helm
 
-Please follow our [Helm Chart Readme](https://github.com/gravitational/teleport/tree/master/examples/chart/teleport) for install and setup instructions.
+Please follow our [Helm guides](kubernetes-access/helm/guides.mdx) for detailed install and setup instructions, including HA setups.
```

Comment:
Nit: stylistically we should prefer `High Availability over `HA` and only use acronyms after we've introduced what it stands for. Do you mind changing this here and below?